src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -695,7 +695,8 @@ export const ComputeModalBase = ({
         currentRuntime ? Ajax().Runtimes.runtime(currentRuntime.googleProject, currentRuntime.runtimeName).details() : null,
         Ajax()
           .Buckets
-          .getObjectPreview(googleProject, getConfig().terraDockerImageBucket, getConfig().terraDockerVersionsFile, true),
+          .getObjectPreview(googleProject, getConfig().terraDockerImageBucket, getConfig().terraDockerVersionsFile, true)
+          .then(r => r.json()),
```

Comment:
Correct. This function gets the contents of an object in GCS. In this case, it's getting a JSON file. But in other cases, it may get a plain text file or binary file.

https://github.com/DataBiosphere/terra-ui/blob/8c7d9d232659fe8ed680a1eddf4a0d71dca71e3d/src/pages/workspaces/workspace/analysis/RuntimeManager.js#L38-L41